app/core/components/Comments.tsx

```diff
@@ -104,7 +104,7 @@ function CommentItem({
           <Avatar alt={"alt"} src={comment.author?.avatarUrl ?? ""}></Avatar>
         </Grid>
         <Grid justifyContent="left" item xs zeroMinWidth>
-          <Typography variant="body1">{`${comment.author?.firstName} ${comment.author?.lastName}`}</Typography>
+          <Typography variant="body1">{`${comment.author?.preferredName} ${comment.author?.lastName}`}</Typography>
```

Comment:
what if the preferred name is the last name? We would have double last name? 🤔

Reply:
🤔 , good question. Like with @LuisTejedaS , right?

Should we instead just use the preferred name?

Reply:
I think we can leave it like it is. That's how Bamboo actually show it:

<img width="475" alt="image" src="https://user-images.githubusercontent.com/15214/226494798-00cd7017-63e3-49a1-8804-12d66e8894c6.png">